src/frontend/src/components/common/Header/Header.tsx

```diff
@@ -129,9 +129,11 @@ function Header({ user }: Props) {
             alt="Go to the Government of British Columbia website"
           />
         </a>
-        <h1 className="header-h1">
-          Digital Privacy Impact Assessment (DPIA) <span>beta</span>
-        </h1>
+        <p className="header-h1">
```

Comment:
This class is removed in this PR

```suggestion
        <p>
```